src/Http/OpenApi/src/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,6 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Microsoft.AspNetCore.OpenApi.Tests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100f33a29044fa9d740c9b3213a93e57c84b472c84e0b8a0e1ae48e67a9f8f6de9d5f7f3d52ac23e48ac51801f1dc950abe901da34d2a9e3baadb141a17c77ef3c565dd5ee5054b91cf63bb3c6ab83f72ab3aafe93d0fc3c2348b764fafb0b1c0733de51459aeab46580384bf9d74c4e28164b7cde247f891ba07891c9d872ad2bb")]
```

Comment:
Suggest getting rid of this file and adding the following to the project file:
``` msbuild
  <ItemGroup>
    <InternalsVisibleTo Include="Microsoft.AspNetCore.OpenApi.Tests" />
  </ItemGroup>
```

Reply:
Good point!

This reminded me of an observation that I had made earlier about inconsistencies between IVT being specified via `AssemblyInfo` and IVT being configured via build props.

I opened https://github.com/dotnet/aspnetcore/issues/41244 to track us being consistent through the repo here in the event we want to take a stand a particular approach.